src/core/sort.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @param {{priority: number}} a - First object with priority property.
+ * @param {{priority: number}} b - Second object with priority property.
+ * @returns {number} A number indicating the relative position.
+ */
```

Comment:
To prevent this from appearing in the API reference manual, you'll need to add `@ignore`.